cmd/kubeless/deploy.go

```diff
@@ -74,7 +89,7 @@ var deployCmd = &cobra.Command{
 			topic = ""
 		}
 
-		err = utils.CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic, ns, deps)
+		err = utils.CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic, ns, deps, description, labels, envs)
```

Comment:
If the eventual goal is to allow just about every part of a Function object to be specified via command line flags, then you should probably construct the Function object here and pass it to `CreateK8sCustomResource` as an argument.